### PR TITLE
Drone heal sunder

### PR DESCRIPTION
**Pull request by BobSlim**
## About The Pull Request

Sunder is a fairly new mechanic, and therefore not well integrated into the game. This adds some amount of healing to sunder for drones and queens. 

## Why It's Good For The Game

Gives drones a little more combat utility - healing sunder enables the healed xenomorph additional time in an extended conflict. 

Also gives queens sunder heal. It's nice thematically that the queen "chooses" her desired warriors to continue fighting. I've also heard talk that the queen is unwanted in comparison to the Shrike. This might change that a bit.

Should the shrike/hivemind have it too? I dunno. Maybe.

## Changelog
:cl:
add: the Queen and Drone heals now heal some sunder.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/xenomorph/castes/drone/abilities_drone.dm [hunk 1]
```diff
@@ -68,10 +68,12 @@
 
 	succeed_activate()
 	add_cooldown()
-
-/mob/living/carbon/xenomorph/proc/salve_healing()	//Slight modification of the heal_wounds proc
+	
+//Slight modification of the heal_wounds proc
+/mob/living/carbon/xenomorph/proc/salve_healing()
```
Comment (spookydonut) [inline code/modules/mob/living/carbon/xenomorph/castes/drone/abilities_drone.dm:73]:
```suggestion

/mob/living/carbon/xenomorph/proc/salve_healing() //Slight modification of the heal_wounds proc
```

Comment (spookydonut) [reply code/modules/mob/living/carbon/xenomorph/castes/drone/abilities_drone.dm:73]: this was fine as it was

Comment (psykzz) [reply code/modules/mob/living/carbon/xenomorph/castes/drone/abilities_drone.dm:73]: If you intended for this to be docs, you could clean up the statement to be clearer.
You also need to use 3 `///`.

**Comment by jroinc** [comment]
Not sure how I feel about this. As proposed, 1 heal would heal 8 sunder, which is a LOT. I agree it needs work, but I'd rather see droneheal heal 2, or even 1. 

Droneheal's intentionally spammable and the entire point of sunder's to grant a lasting debuff. With it healing 8, 1-2 heals would eliminate all sunder normally obtainable

**Pull request merged**